### PR TITLE
Add testing and support for Django 2.0.

### PR DESCRIPTION
**Pull request by jdufresne**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 django-storages change log
 ==========================
 
+UNRELEASED
+**********
+
+* Add support for Django 2.0.
```
Comment (timgraham) [inline CHANGELOG.rst]: You might say "Add testing for Django 2.0 (no code changes were required)."

**Comment by jdufresne** [comment]
Thanks for the review. I made the suggested wording changes to the change log and commit message. I also updated the tox requirement to `Django>=2.0rc1, <2.1`

**Comment by jdufresne** [comment]
> it might also be an opportunity to drop support for Django < 1.11

I've gone ahead and added a commit to do so. I agree in doing so.

@jschneier If you disagree with the approach, please let me know and I can remove the commit.

The lint error is fixed in #415

**Comment by jdufresne** [comment]
This was fixed in cecb101a8eacf3dda708e251471890941fc994a8

**Pull request closed without merging**